buildscripts/condarecipe.local/meta.yaml

```diff
@@ -35,6 +35,7 @@ requirements:
     - llvmlite 0.31.*
     - funcsigs                 # [py27]
     - singledispatch           # [py27]
+    - importlib-metadata       # [py27]
```

Comment:
I am not sure this is needed still as we only support 3.6 and newer. The two lines above are being removed in https://github.com/numba/numba/pull/5210/files

Reply:
I updated selector to `[py<38]`, it is still needed for py36 and py37.

Reply:
sounds good.